### PR TITLE
Update Windows install instructions to target the py launcher

### PR DESCRIPTION
**Pull request by johnthagen**
# Pull Request Check List

Resolves: #5611

### CODE REVIEW

**Comment by johnthagen** [comment]
@abn Could you review this?

**Review by Secrus — APPROVED** [review]

File: docs/_index.md [hunk 1]
```diff
@@ -43,9 +43,14 @@ curl -sSL https://install.python-poetry.org | python3 -
 
 **windows powershell install instructions**
 ```powershell
-(Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | python -
+(Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | py -
```
Comment (abn) [inline docs/_index.md:46]: Should it be the other way around? python by default and note to suggest use of py?

Also does Windows have an equivalent of `py || python`? Not even sure if that would work.

Comment (abn) [reply docs/_index.md:46]: Just saw your comment on the issue.

Comment (Secrus) [reply docs/_index.md:46]: `py` is default runner when installing from `python.org`.

**Review by abn — APPROVED** [review]

**Pull request merged**